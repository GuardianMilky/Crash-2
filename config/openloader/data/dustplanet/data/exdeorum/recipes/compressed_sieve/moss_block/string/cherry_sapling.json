{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/moss_block"},"mesh":"exdeorum:string_mesh","result":"minecraft:cherry_sapling","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.11}}