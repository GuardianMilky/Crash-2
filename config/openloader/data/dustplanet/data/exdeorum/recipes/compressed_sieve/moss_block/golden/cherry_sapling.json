{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/moss_block"},"mesh":"exdeorum:golden_mesh","result":"minecraft:cherry_sapling","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.11}}